Microsoft.Toolkit.Uwp.SampleApp/Assets/Photos/ImageCache.json

```diff
@@ -0,0 +1,30 @@
+﻿[
+  "https://invokeit.files.wordpress.com/2016/09/img_1326.jpg",
```

Comment:
My idea was to test with a real life data set of something I'd use it for. Originally I used images from a publicly available article with images in public domain being pulled from json file. After discussion with @ScottIsAFool I uploaded a smaller set of personal images.

The existing dataset wasn't large enough to form a conclusive test.